partitioned-table.md

```diff
@@ -724,9 +724,12 @@ You can see that the inserted record `(NULL, 'mothra')` falls into the same part
 
 ## Partition management
 
-For `LIST` and `RANGE` partitioned tables, you can add and drop partitions using the `ALTER TABLE <table name> ADD PARTITION (<partition specification>)` or `ALTER TABLE <table name> DROP PARTITION <list of partitions>` statement.
+For `LIST` and `RANGE` partitioned tables, you can manage the partitions as follows:
```

Comment:
```suggestion
For List/List COLUMNS/Range/Range COLUMNS partitioned tables, you can manage the partitions as follows:
```
IMO we distinguished List/List COLUMNS the definition early in this article, it's better to clarify them here, so do Range/Range COLUMNS.